README.md

```diff
@@ -30,14 +30,71 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The .forEach doesn't return anything.  For each array element it will call the function, and when it's finished, it's done.  I think .forEach is a good general tool if touching each element in an array is important.
+
+The .map also calls the function and passes through each array element, but it will build a new array with the values.  Map is great for changing elements in an array. For instance, in the case of numbers, multiplying each number by something like 3 and returning a new array with the changes.
+
+
 2. What is the difference between a function and a method?
 
+  A method is a function inside of an object, or should I say within the scope of an object.  So I guess one could say a method works implicitly inside the object it is called in.
+
+  A function is called by name.  Perhaps one could say a method is too, but I think the difference is that a function doesn't have  an object associated with it. 
+
```

Comment:
Yes. Don't forget you can have an anonymous function. I would say the main difference is a method belongs to an object